administrator/components/com_templates/Model/TemplateModel.php

```diff
@@ -309,14 +307,7 @@ private function validateDate($date)
 	 */
 	private function findPath($paths, $file)
 	{
-		$app = Factory::getApplication();
-
-		if (!$path = Path::find($paths, $file))
-		{
-			$app->enqueueMessage(Text::_('COM_TEMPLATES_ERROR_CORE_FILE_NOT_FOUND'), 'warning');
-		}
-
-		return $path;
+		return Path::find($paths, $file);
```

Comment:
Do we really need our own method for this one line?

Reply:
Agree

Reply:
I also agree. I fix this.